index.html

```diff
@@ -118,149 +92,150 @@ <h2>
-      // object is guaranteed to be at most 10 minutes old.
-    }
+      function successCallback(position) {
+        // By using the 'maximumAge' option above, the position
+        // object is guaranteed to be at most 10 minutes old.
+      }
 
-    function errorCallback(error) {
-      // Update a div element with error.message.
-    }
-    </pre>
-      <pre class="example" title=
-      'Forcing the user agent to return a fresh cached position'>
-    // Request a position. We only accept cached positions whose age is not
-    // greater than 10 minutes. If the user agent does not have a fresh
-    // enough cached position object, it will immediately invoke the error
-    // callback.
-    navigator.geolocation.getCurrentPosition(successCallback,
-                                             errorCallback,
-                                             {maximumAge:600000, timeout:0});
+      function errorCallback(error) {
+        // Update a div element with error.message.
+      }
+      </pre>
+      <pre class="example js" title=
+      "Forcing the user agent to return a fresh cached position">
+      // Request a position. We only accept cached positions whose age is not
+      // greater than 10 minutes. If the user agent does not have a fresh
+      // enough cached position object, it will immediately invoke the error
+      // callback.
+      navigator.geolocation.getCurrentPosition(successCallback, errorCallback, {
+        maximumAge: 600000,
+        timeout: 0,
+      });
 
-    function successCallback(position) {
-      // By using the 'maximumAge' option above, the position
-      // object is guaranteed to be at most 10 minutes old.
-      // By using a 'timeout' of 0 milliseconds, if there is
-      // no suitable cached position available, the user agent
-      // will asynchronously invoke the error callback with code
-      // TIMEOUT and will not initiate a new position
-      // acquisition process.
-    }
+      function successCallback(position) {
+        // By using the 'maximumAge' option above, the position
+        // object is guaranteed to be at most 10 minutes old.
+        // By using a 'timeout' of 0 milliseconds, if there is
+        // no suitable cached position available, the user agent
+        // will asynchronously invoke the error callback with code
+        // TIMEOUT and will not initiate a new position
+        // acquisition process.
+      }
 
-    function errorCallback(error) {
-      switch(error.code) {
-        case error.TIMEOUT:
-          // Quick fallback when no suitable cached position exists.
-          doFallback();
-          // Acquire a new position object.
-          navigator.geolocation.getCurrentPosition(successCallback, errorCallback);
-          break;
-        case ... // treat the other error cases.
-      };
-    }
+      function errorCallback(error) {
+        switch (error.code) {
+          case GeolocationPositionError.TIMEOUT:
```

Comment:
Nit: the existing syntax is more readable IMO. Both are correct. A matter of taste I guess. If there's a convention for this in other specs, let's follow that.

Reply:
The idea is to show the that the static constant in use.